packages/ember-glimmer/lib/component-managers/curly.ts

```diff
@@ -412,6 +412,18 @@ export function validatePositionalParameters(named: NamedArguments, positional:
 }
 
 export function processComponentInitializationAssertions(component: Component, props: any) {
+  assert(`classNameBindings must be strings: ${component.toString()}`, (() => {
```

Comment:
Seems fine, but I would have assumed that the explicit `.toString()` shouldn't have been needed inside template strings.  Mind confirming?

Reply:
Yep, that's the case. I've updated the other similar tests to remove `toString` and use template strings too